contrib/ros/README.md

```diff
@@ -1,199 +1,38 @@
-# rllab.contrib.ros Instructions
-## Introduction
-Most of the current environments for reinforcement learning training(e.g. gym's environments) are not particularly 
-designed for robot learning. Consequently, most of their environments are not for continuous action space task. It is also hard for robot learning researchers to use the same environment interface when they want to test their 
-algorithms on real robots. 
+# Rllab Robotics Extension called rllab-ros
+## Reinforcement Learning with ROS and Gazebo
+The Rllab is a toolkit for reinforcement learning which 
```

Comment:
Rllab is a toolkit for reinforcement learning that has recently gained popularity in the robot learning community.

The work presented here builds a general interface for the ros environment that combines robot learning in simulation and in the real world.